### PR TITLE
Serverside sorting

### PR DESCRIPTION
**Pull request by chowington**
Works toward https://github.com/VEuPathDB/web-eda/issues/1176

### CODE REVIEW

File: src/components/grids/DataGrid/index.tsx [hunk 1]
```diff
@@ -200,6 +204,8 @@ export default function DataGrid({
     }
   );
 
+  const finalSortBy = useMemo(() => sortBy.filter((column) => column.desc !== undefined), [sortBy]) as SortBy;
```
Comment (asizemore) [inline src/components/grids/DataGrid/index.tsx:207]: suggestion - maybe "filteredSortBy" would be more clear? Or perhaps a comment on why we need to filter would be helpful! I'd have assumed that sortBy only included columns that had a defined .desc value.

**Review by asizemore — COMMENTED** [review]
Works great! I noticed when i do Shift+click to multisort, i end up always highlighting the column names. Would changing it to ctrl+click avoid the issue (if we even have that control)?

Screen Shot 2022-07-22 at 3 55 22 PM

**Review by asizemore — APPROVED** [review]
Looks great! Thanks @chowington !!

**Comment by chowington** [comment]
Thanks for reviewing, Ann!

**Pull request merged**